src/fparser/common/tests/utf.f90

```diff
@@ -0,0 +1,4 @@
+program utf
```

Comment:
Please add license/copyright.

Reply:
OK, done. I've also cut down the example.